src/components/WMSLayersPanel.vue

```diff
@@ -71,25 +71,21 @@ export default {
     },
     //filter layer based on current epsg
     filterLayerByCurrentEpsg(){
-      this.layers = this.layers.filter(({name}) => this.layerProjections[name].crss.indexOf(this.epsg) !== -1);
+      this.layers = this.epsg !== null ? this.layers.filter(({name}) => this.layerProjections[name].crss.indexOf(this.epsg) !== -1) : this.$options.config.layers;
     }
   },
   watch: {
-    // when chanhe selected layers
+    // when change selected layers
     selectedlayers(layers){
       if (layers.length) {
         const firstLayer = layers[0];
         if (layers.length === 1) {
           this.epsg = this.layerProjections[firstLayer].crss[0];
-          this.projections = this.layerProjections[firstLayer].crss;
+          this.projections = [...this.layerProjections[firstLayer].crss];
```

Comment:
@volterra79 Can you add comment above this line? (eg. shallow copy required due to ...)

Reply:
@Raruto i add new commit without shallow copy